### PR TITLE
Remove unused PayClientBindingSwg

### PR DESCRIPTION
**Pull request by alin04**
- Remove unused PayClientBindingSwg.

gpay experiment has been at 100% on SwG and AMP-swg for awhile now.

### CODE REVIEW

File: src/runtime/pay-client.js [hunk 1]
```diff
@@ -342,13 +109,33 @@ export class PayClientBindingPayjs {
...
+   */
   getType() {
+    // TODO(alin04): remove once all references removed.
     return 'PAYJS';
   }
 
-  /** @override */
-  start(paymentRequest, options) {
+  /**
+   * @param {!Object} paymentRequest
```
Comment (mborof) [inline src/runtime/pay-client.js]: Can we make payment request a specific kind of object?

File: src/runtime/pay-client.js [hunk 2]
```diff
@@ -386,7 +173,9 @@ export class PayClientBindingPayjs {
     return promise;
   }
 
-  /** @override */
+  /**
+   * @param {function(!Promise<!Object>)} callback
```
Comment (mborof) [inline src/runtime/pay-client.js]: Can we make this Promise return a specific kind of object?

Comment (alin04) [reply src/runtime/pay-client.js]: Done.

File: src/runtime/pay-client.js (hunk 1 above)
Comment (alin04) [reply src/runtime/pay-client.js]: Done

**Pull request merged**